raudelb/session08/circle_test.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+from session08.circle import Circle
+
+
+class CircleTest(unittest.TestCase):
```

Comment:
Great job on making tests for the circle class.  I ran you test through `coverage` which is a module that assess what lines in your program are missed in your tests.  you can install yourself via:
`pip install coverage`
and then run in the directory where your tests live:
`coverage run test_circle.py`
and then:
'coverage report'

If you are running pycharm, you can right click anywhere within you test py and select Run with coverage and it will highlight the lines you covered and missed.  Based on that, these two functions were missed in your tests:

```
@property
    def diameter(self):
        return 2*self.radius

    @diameter.setter
    def diameter(self, value):
        self._diameter = value
        self.radius = value/2
```